ops/pebble.py

```diff
@@ -3052,6 +3109,28 @@ def get_checks(
         resp = self._request('GET', '/v1/checks', query)
         return [CheckInfo.from_dict(info) for info in resp['result']]
 
+    def start_checks(self, checks: Iterable[str]) -> FrozenSet[str]:
+        """Start checks by name.
+
+        Args:
+            checks: Non-empty list of checks to start.
+
+        Returns:
+            Set of check names that were started.
```

Comment:
Can we clarify with something like "Checks that were already started won't appear in this list."? (Here and in model.py)